tests/filters.rs

```diff
@@ -211,3 +211,31 @@ pub fn append() {
     let output = template.render(&mut data);
     assert_eq!(output.unwrap(), Some("roobarblifo".to_string()));
 }
+
+#[test]
+// Got this test from example at https://shopify.github.io/liquid/filters/split/
+// This is an additional test to verify the comma/space parsing is also working
+// from https://github.com/cobalt-org/liquid-rust/issues/41
+pub fn split_with_comma() {
+    let text = "{% assign beatles = \"John, Paul, George, Ringo\" | split: \", \" %}{% for member in beatles %}{{ member }}\n{% endfor %}";
```

Comment:
I had it as single quotes originally, but I changed when #41 was still a thing.
Strictly speaking, these don't need to be double quotes.

Do you want them changed?  Doesn't matter to me...  Seems like there are already plenty of tests that do single quotes but none that use double quotes.  Sometimes helps to see examples...

Up to you.